Makefile

```diff
@@ -1,26 +1,30 @@
-CFLAGS=		-g -Wall -O2 -Wc++-compat #-Wextra
+CFLAGS=		-g -Wall -O2 -Wc++-compat  #-Wextra
```

Comment:
Unnecessary difference. 1 space here.

Reply:
I've removed the space in the following commit.